aquarius/events/processors.py

```diff
@@ -198,6 +200,12 @@ def make_record(self, data):
 
     def process(self):
         txid = self.txid
+
+        dt_factory = get_dt_factory(self._web3)
+        if dt_factory.caller.erc721List(self.event.address) != self.event.address:
+            logger.info("token not deployed by our factory")
```

Comment:
maybe logger.error?   just to make sure that whoever looks in the logs, will see it